_layouts/cv.liquid

```diff
@@ -54,12 +54,17 @@ layout: default
         {{ page.title }}
         {% if page.cv_pdf %}
           <a
-            href="{{ page.cv_pdf | prepend: 'assets/pdf/' | relative_url}}"
+            {% if page.cv_pdf contains '://' %}
+              href="{{ page.cv_pdf }}"
+            {% else %}
+              href="{{ page.cv_pdf | prepend: 'assets/pdf/' | relative_url}}"
```

Comment:
Just one more thing, can you add a space between `relative_url` and `}}`?

Reply:
Sure, done!